src/utils.js

```diff
@@ -1,11 +1,5 @@
 import {adoptedSheetsRegistry, frame, OldCSSStyleSheet} from './shared';
-
-export function instanceOfStyleSheet(instance) {
-  return (
-    instance instanceof OldCSSStyleSheet ||
-    instance instanceof frame.CSSStyleSheet
-  );
-}
+import { instanceOfStyleSheet } from './ConstructStyleSheet';
```

Comment:
No need to change this function location.

Reply:
Leaving it in the old location creates a circular reference between `utils.js` and `ConstructStyleSheet.js`. It also feels more natural to have the `instanceOf` logic living close to the class it referencing (IMO). The implementation of the method might also need updating because we should not really be returning "true" for instances of the browser default class (referred to as OldCssStyleSheet throughout code).

Reply:
OK, I'll accept that move in that case. It's not really a problem either way, but avoiding circular dependencies is always a good idea.

We do need that check to make sure that any element passed to the adopted stylesheets array is an instance of the browser's `CSSStyleSheet`, which the `Symbol.hasInstance` takes care of for us. I'l resolve this conversation.